src/spring/azext_spring/tests/latest/test_asa_scenario.py

```diff
@@ -218,12 +218,3 @@ def test_app_deploy_container(self):
                  + ' --container-image {containerImage} --registry-username PLACEHOLDER --registry-password PLACEHOLDER', checks=[
             self.check('name', 'green'),
         ])
-
-        with self.assertRaisesRegexp(CLIError, "Failed to wait for deployment instances to be ready"):
-            self.cmd('spring app deploy -g {resourceGroup} -s {serviceName} -n {app} --artifact-path {file}')
-
-        self.cmd('spring app deployment show -g {resourceGroup} -s {serviceName} -n default --app {app} ', checks=[
-            self.check('name', 'default'),
-            self.check('properties.source.type', 'Jar'),
-            self.check('properties.source.runtimeVersion', 'Java_8'),
```

Comment:
Why did you remove these tests?

Reply:
Because it's meaningless for CustomImageTest. These commands are not used to deploy app with customer image but jar.